main.html

```diff
@@ -5,9 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Fede's Portfolio</title>
-    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;700&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&family=Splash&display=swap" rel="stylesheet">
 </head>
 <body>
-    
+    <header>
+        <div class="top">
+            <div class="logo">
+                <i class="fa fa-moon-o"></i> <span class="logo-name">Luna</span>
```

Comment:
- [ ] @Flunavgt great work, the previous reviewer forgot to indicate that the logo needs to have an `<a>` tag, it's suppose to be clickable, my apologies on the reviewer's behalf.

Reply:
- [ ] Kindly check on your design, I cant see the background image, also ensure your design matches the Figma design given, below is a comparison:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/176731323-823a4bc9-4e5e-4bb3-975b-d9ba6eddce5c.png) |  ![image](https://user-images.githubusercontent.com/61408860/176731636-d5514b9c-4f74-498a-b71c-b9fdb3fa0d01.png)